riscv/tests/riscv.rs

```diff
@@ -628,6 +690,7 @@ fn profiler_sanity_check() {
     assert!(!callgrind.unwrap().is_empty());
 }
 
+#[cfg(feature = "plonky3")]
```

Comment:
why does it need ploky3 now?

Reply:
It already needed, but without this switch, build fails if plonky3 is not activated in the backend crate (and it isn't, by default).

Reply:
These two changes in the features you commented is fixing unrelated issues I found while trying to debug the test I wrote inside the IDE. The building of the tests was failing when building just the crate, turns out because required features were not being enabled.